### ISSUE TITLE
Server lag on world save

### ISSUE THREAD

**Issue by Coach-mdl** [TeamAOF/Craftoria#210]
I'm not entirely sure if this is just a me issue since my system isn't exactly the strongest. 

Whenever the world saves, be it auto save or pausing, the world freezes for a moment. When pausing it's not an issue. When you're fighting a horde of monsters and then the game decides to pop a save, it's the equivalent of the game going "ZA WARUDO!" And completely running you into the ground.

This is the first 1.21 modpack I've played so I'm not sure if this is simply a 1.21 issue or if there's something that can be done to help with this. My current remedy is pausing the game and waiting for it to finish saving, but it's annoying to do every 5 minutes.

**Comment by Dragonex97** [comment]
it's not just you, it happens to me too xd

**Comment by Shimon-z** [comment]
jave you guys found any way to fix it?

**Comment by Coach-mdl** [comment]
Not that I'm aware of, but I also stopped playing due to this. This is partially a hardware related issue. Seeing the pack run on a better system nulls this issue completely. I also haven't had much experience with 1.21 packs so I don't know if this is potentially a base game issue or a mod loader related one. Either way, the game runs smoothly right until the game decides to save.

**Comment by SubordinalBlue** [comment]
Yeah, this is a weird one.. world saving in a vanilla feature and the pack has no mods that modify that, afaik.

You're not the only one to report it, but its been an annoyingly small minority... annoying as in so far frustratingly hard to diagnose what's really going on.

**Comment by Gronglegrowth** [comment]
I think I remember seeing discussions that it could be related to playing on an HDD - a comparatively much slower storage device than an SSD?

**Comment by SubordinalBlue** [comment]
Good point; what's your storage situation, OP?

**Comment by Coach-mdl** [comment]
Exactly that. A HDD. Not to mention quite an old one. Got the system back in 2018 ish and it was not a future proofed one. Thing still uses DDR3 memory.

**Comment by Gronglegrowth** [comment]
It's likely to be that then, I think.

**Comment by Coach-mdl** [comment]
If only there was a way to change the frequency and occurrences of auto saves. I believe it's every 5 mins. Not to mention pausing also auto saves. Auto saving isn't a new feature and it's worked fine on previous versions with a HDD so what happened in 1.21 to change that?

**Comment by Gronglegrowth** [comment]
It could be that there is simply a lot of data being saved - this isn't a small modpack, after all.

**Comment by Shimon-z** [comment]
> It could be that there is simply a lot of data being saved - this isn't a small modpack, after all.

i have the same problem, but i also played ATM series and there my game runned fine.. and not just ATM, i've played countless mods in version 1.21, which means this problem happens specifically in craftoria
